src/object-cache.cls.php

```diff
@@ -146,19 +146,18 @@ public function __construct($cfg = false)
 	 * @since  6.3
 	 * @access private
 	 */
-	private function debug_oc($text, $show_error = false)
+	private function debug_oc($text)
 	{
-		if (defined('LSCWP_LOG')) {
-			Debug2::debug($text);
-
+		if (!$this->_cfg_debug) {
 			return;
 		}
 
-		if (!$show_error && !$this->_cfg_debug) {
-			return;
-		}
+		// For initiate Debug2 class
+		!defined('LITESPEED_DATA_FOLDER') && define('LITESPEED_DATA_FOLDER', 'litespeed');
+		!defined('LITESPEED_STATIC_DIR') && define('LITESPEED_STATIC_DIR', WP_CONTENT_DIR . '/'. LITESPEED_DATA_FOLDER);
+		$debug2 = new Debug2();
```

Comment:
You can't initialize in all the calls. This is low efficient.

Reply:
Fixed by moving the initialization to constructor. 👍